code/modules/projectiles/guns/energy/kinetic_accelerator.dm

```diff
@@ -230,6 +230,8 @@
 	var/obj/effect/temp_visual/kinetic_blast/K = new /obj/effect/temp_visual/kinetic_blast(target_turf)
 	K.color = color
 
+/obj/projectile/kinetic/mech/ //mecha_kineticgun version of the projectile
```

Comment:
```suggestion
//mecha_kineticgun version of the projectile
/obj/projectile/kinetic/mech
```
Avoid a `/` at the end of the path, and careful about comments after it.
we've had some funky types resulting from this pattern.